src/formatDate.js

```diff
@@ -50,7 +50,76 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const currentSep = fromFormat[3];
```

Comment:
what is sep?

Reply:
I wanted to name it currentSeperator but then it was gonna be very long.

Reply:
it is much better to keep it long but clear than to have short unclear variables